.github/workflows/build-container.yml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     branches:
       - main
+    tags:
+      - '*'
```

Comment:
Line 39 will also need to change, so that `latest` is not always used as the tag.

[This setup](https://github.com/brown-bnc/xnat-tools/blob/b35453575843d82c77cd7d6ba05348ff44cfd917/.github/workflows/publish.yml#L73) has the labels that I think you want

Reply:
I made the change here. On the BKE side, would I make the change in the `dockerconfig.enc.json`?